var/spack/repos/builtin/packages/numactl/package.py

```diff
@@ -0,0 +1,39 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Numactl(AutotoolsPackage):
+    """NUMA support for Linux"""
+
+    homepage = "http://oss.sgi.com/projects/libnuma/"
+    url      = "https://github.com/numactl/numactl/archive/v2.0.11.tar.gz"
```

Comment:
The official download location ftp://oss.sgi.com/www/projects/libnuma/download/ has the tarballs with `configure` in it, so you could drop the dependencies below.

Reply:
thanks!
I am unsure which one to prefer, since the [official homepage](http://oss.sgi.com/projects/libnuma/) states:
```
Download of Source Code

The numactl command and libnuma library are now maintained at https://github.com/numactl/numactl. 
```

which I would also think is valid for releases?

Reply:
@junghans the offical downloads in `ftp://oss.sgi.com/www/projects/libnuma/download/` have again a nasty `old/` structure for older releases which will keep us running to update the versions that are "supported" and unsupported ("old") within this package is in the `libsodium` package: #5355

I would therefore propose to use the git tags since they are easier to maintain.